deploy/vagrant/provision/start_tachyon_cluster.sh

```diff
@@ -2,5 +2,5 @@
 
 cd /tachyon/bin
 ./tachyon format
-./tachyon-start.sh all Mount
+./tachyon-start.sh all SudoMount
```

Comment:
when I haven't changed the commit hash in tachyon_version.yml, the corresponding tachyon version has permission issues on my mac when mount without sudo, the jira issue, you remember? 

yes, it should be removed now.